lib/restrictions.js

```diff
@@ -176,7 +203,7 @@ export async function get({zone, resource} = {}) {
       'Restriction not found.',
       'NotFoundError', details);
   }
-  return record;
+  return {restrictions: records};
```

Comment:
This is also a departure from what we usually do -- and we're returning full records here, not restrictions. I'm ok with us returning `{records}` if we want to allow for adding other properties later. Otherwise, we should just return `records`.

```suggestion
  return {records};
```

Reply:
Updated.

Reply:
Correction, the reason this returned an object instead of an array is because something we use downstream to call this does not handle being passes arrays, it prefers objects. Updating again.